serverless/src/getContactInfo/handler.js

```diff
@@ -65,6 +68,11 @@ const getContactInfo = async (event, context) => {
       body: JSON.stringify(contactInfoData)
     }
   } catch (e) {
+    console.log({
```

Comment:
We don't need this console.log here, in all the other lambdas we just return the error formatted by `parseError`, that in turn gets reported to the `errorLogger` lambda for logging